lms/djangoapps/learner_dashboard/programs.py

```diff
@@ -91,6 +93,14 @@ def _get_resource_link_id(program_uuid, request) -> str:
     def _get_result_sourcedid(context_id, resource_link_id, user_id) -> str:
         return f'{context_id}:{resource_link_id}:{user_id}'
 
+    def get_user_roles(self, user):
+        """
+        Returns the Administrator role if the user is Global staff else returns the Student role
```

Comment:
the docstring is describing code, which it shouldn't. The docstring should explain the behavior. 
e..g. `Returns the given user role.`